com.unity.render-pipelines.universal/Editor/Camera/URPSerializedCamera.cs

```diff
@@ -0,0 +1,62 @@
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    internal class URPSerializedCamera
+    {
+        internal SerializedObject serializedObject;
```

Comment:
All the `internal` members (properties, constructors, fields included) should be `public` when the type is `internal`. Also, this one should probably also be a get-only property. If not that, then a readonly field, as changing the value would leave it inconsistent with all the other values.